src/main/java/training/Shapes

```diff
@@ -0,0 +1,70 @@
+package training;
+
+ class Area
+{
+    public static void main(String[] args) {
```

Comment:
Now we need to do some cleaning, instead of creating main function here, this class should have one public method getAreaforAllShapes() there should not be any main method here.